ui-frontend/packages/catalog-ui-search/src/main/webapp/component/metacard-overwrite/metacard-overwrite.view.js

```diff
@@ -211,7 +211,7 @@ class MetacardOverwrite extends React.Component {
   constructor(props) {
     super(props)
     this.state = defaultState
-    this.model = store.getSelectedResults().first()
+    this.model = SelectionInterface.getSelectedResults().first()
```

Comment:
For this to work, I think you need to pass an instantiated selection interface model from the parent component. I don't think you can globally access `SelectionInterface` like this

Reply:
You're right, I was having trouble testing this one but as soon as I got it running I realized this wouldn't work.